modules/authentication/containers/ForgotPassword.js

```diff
@@ -0,0 +1,21 @@
+import {
+  connect
+} from 'react-redux';
+import {
+  bindActionCreators
+} from 'redux';
+import * as authenticationActions from '../actions';
+import ForgotPassword from '../components/ForgotPassword';
+
+function mapDispatchToProps(dispatch) {
+  return {
+    actions: {
+      authentication: bindActionCreators(authenticationActions, dispatch)
```

Comment:
I definitely like it. I still don't understand the magic behind it but I have yet to encounter a problem.